glommio/src/executor/mod.rs

```diff
@@ -1140,6 +1141,9 @@ impl LocalExecutor {
         cpu_binding: Option<impl IntoIterator<Item = usize>>,
         mut config: LocalExecutorConfig,
     ) -> Result<LocalExecutor> {
+        let blocking_thread =
+            BlockingThreadPool::new(config.thread_pool_placement, notifier.clone())?;
```

Comment:
So technically this now maybe has problems with the NUMA logic below? Not 100% sure about the impact.

The other way to do this maybe is to retrieve the CPU affinity prior to binding & passing that into `BlockingThreadPool` so that it resets the CPU affinity when the thread starts?

Reply:
I think this should be fine. The executor is still allocated in the node where it wants to run, and the BlockingThreadPool is way less performance sensitive.